lib/extensions/require_nested.rb

```diff
@@ -15,6 +15,14 @@ def require_nested(name)
       Object.require_nested name
     end
   end
+
+  def require_nested_all(path)
+    path = "#{path[0...-3]}/*.rb" # i.e. 'mod/abc.rb' => 'mod/abc/*.rb'
```

Comment:
I don't understand the 0...-3. What is that when talking about a path?

Reply:
Oh I see you are just removing the .rb... Better way to do that is...

    path.chomp(".rb")

Reply:
i just figure that anything doing a string comparison would be slower than index calculation...